app/models/viewhelper/ProjectData.scala

```diff
@@ -73,44 +76,53 @@ object ProjectData {
       lastVisibilityChange,
       lastVisibilityChangeUser)
 
+    //TODO: Why Future here?
```

Comment:
maybe to have to same usage as `def of[A](project: Project)` but we don't need this here I think